apps/condo/domains/notification/utils/clientSchema/Message.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Generated by `createschema notification.Message 'organization?:Relationship:Organization:CASCADE; property?:Relationship:Property:CASCADE; ticket?:Relationship:Ticket:CASCADE; user:Relationship:User:CASCADE; type:Text; meta:Json; channels:Json; status:Select:sending,planned,sent,canceled; sendAt:DateTimeUtc;'`
+ */
+
+import { pick, get } from 'lodash'
+
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
+import { generateReactHooks } from '@condo/domains/common/utils/codegeneration/generate.hooks'
+
+import { Message as MessageGQL } from '@condo/domains/notification/gql'
+import { Message, MessageUpdateInput, QueryAllMessagesArgs } from '../../../../schema'
+
+const FIELDS = ['id', 'deletedAt', 'createdAt', 'updatedAt', 'createdBy', 'updatedBy', 'organization', 'property', 'ticket', 'user', 'type', 'meta', 'channels', 'status', 'sendAt']
+const RELATIONS = ['organization', 'property', 'ticket', 'user']
+
+export interface IMessageUIState extends Message {
+    id: string
+    // TODO(codegen): write IMessageUIState or extends it from
+}
+
+function convertToUIState (item: Message): IMessageUIState {
+    if (item.dv !== 1) throw new Error('unsupported item.dv')
+    return pick(item, FIELDS) as IMessageUIState
+}
+
+export interface IMessageFormState {
+    id?: undefined
```

Comment:
```suggestion
    id?: void
```

Reply:
`void` is assignable to null || undefined
And `null` value is not correct in this case